src/logs/metadata_engine/metadata_engine.py

```diff
@@ -115,7 +118,7 @@ def _load_configs(self):
                         self.default_rule = _create_config_rules(config_json)[0]
                     else:
                         self.rules.extend(_create_config_rules(config_json))
-            except Exception:
+            except Exception as ex:
                 logging.exception(f"Failed to load configuration file: '{config_file_path}'")
```

Comment:
ex not added to log here

Reply:
If you look in the implementation of `logging.exception()`, the exception details are retrieved using `traceback`: https://docs.python.org/3/library/traceback.html

Reply:
Sounds good, but then why this "as ex" part? I think it is not needed in such case

Reply:
If it's helpful for you then it's cool, I thought it's some leftover